src/commands/ImportObjectsCommandGenerator.js

```diff
@@ -59,50 +59,59 @@ module.exports = class ImportObjectsCommandGenerator extends BaseCommandGenerato
 	}
 
 	async _getCommandQuestions(prompt) {
+						
+		let listObjectAnswers;
 		try {
 			const listObjectQuestions = this._generateListObjectQuestions();
-			const listObjectAnswers = await prompt(listObjectQuestions);
-
-			const paramsForListObjects = this._arrangeAnswersForListObjects(listObjectAnswers);
-			const executionContextForListObjects = new SDKExecutionContext({
-				command: this._listObjectsMetadata.name,
-				params: paramsForListObjects,
-				includeProjectDefaultAuthId: true,
-			});
+			listObjectAnswers = await prompt(listObjectQuestions);
+		} catch (error) {
+			throw TranslationService.getMessage(PROMPTING_INTERACTIVE_QUESTIONS_FAILED, NodeUtils.lineBreak, error);
+		}			
 
-			let operationResult;
-			try {
-				operationResult = await executeWithSpinner({
-					action: this._sdkExecutor.execute(executionContextForListObjects),
-					message: TranslationService.getMessage(MESSAGES.LOADING_OBJECTS),
-				});
-			} catch (error) {
-				throw TranslationService.getMessage(ERRORS.CALLING_LIST_OBJECTS, NodeUtils.lineBreak, error);
-			}
+		const paramsForListObjects = this._arrangeAnswersForListObjects(listObjectAnswers);
+		const executionContextForListObjects = new SDKExecutionContext({
+			command: this._listObjectsMetadata.name,
+			params: paramsForListObjects,
+			includeProjectDefaultAuthId: true,
+		});
 
-			const { data } = operationResult;
-			if (SDKOperationResultUtils.hasErrors(operationResult)) {
-				SDKOperationResultUtils.logResultMessage(operationResult);
-				SDKOperationResultUtils.logErrors(operationResult);
-				return;
-			}
-			if (Array.isArray(data) && operationResult.data.length === 0) {
-				NodeUtils.println(TranslationService.getMessage(MESSAGES.NO_OBJECTS_TO_LIST), NodeUtils.COLORS.RESULT);
-				return;
-			}
+		let listObjectsResult;
+		try {
+			listObjectsResult = await executeWithSpinner({
+				action: this._sdkExecutor.execute(executionContextForListObjects),
+				message: TranslationService.getMessage(MESSAGES.LOADING_OBJECTS),
+			});
+		} catch (error) {
+			throw TranslationService.getMessage(ERRORS.CALLING_LIST_OBJECTS, NodeUtils.lineBreak, error);
+		}
 
-			const selectionObjectQuestions = this._generateSelectionObjectQuestions(operationResult);
-			const selectionObjectAnswers = await prompt(selectionObjectQuestions);
+		const { data } = listObjectsResult;
+		if (SDKOperationResultUtils.hasErrors(listObjectsResult)) {
+			SDKOperationResultUtils.logResultMessage(listObjectsResult);
+			SDKOperationResultUtils.logErrors(listObjectsResult);
+			throw SDKOperationResultUtils.getErrorMessagesString(listObjectsResult);
+		}
+		if (Array.isArray(data) && listObjectsResult.data.length === 0) {
+			NodeUtils.println(TranslationService.getMessage(MESSAGES.NO_OBJECTS_TO_LIST), NodeUtils.COLORS.RESULT);
+			return;
```

Comment:
If we specify a scriptid string that matches nothing in our account during the interactive questions, the flow of execution won't be stopped here, and will return undefined. Consequently we will have as result: 
```
There are no objects matching your search criteria. Select a different set of object types, and if applicable, check the specified application ID or script ID.
There are validation errors:
"scriptid" option is mandatory.
"type" option is mandatory.
"destinationfolder" option is mandatory.
```

Reply:
To stop the execution flow we need to 'throw' . So I think we need to just:
```
throw NodeUtils.println(TranslationService.getMessage(MESSAGES.NO_OBJECTS_TO_LIST);
```
It will show a red message (which is nice because we could not perform the import) and stop the execution flow,